src/config/localStorage.ts

```diff
@@ -25,6 +25,7 @@ export const KEEP_LEVERAGE_FOR_DECREASE_KEY = "Exchange-keep-leverage";
 export const TRADE_LINK_KEY = "trade-link";
 export const SHOW_DEBUG_VALUES_KEY = "show-debug-values";
 export const ORACLE_KEEPER_INSTANCES_CONFIG_KEY = "oracle-keeper-instances-config";
+export const DEBUG_ACCOUNT_ADDRESS = "debug-account-address";
```

Comment:
hm, we could use it in `useWallet` then it could be a legit flag

but we somewhere use `useAccount` directly, so won't work in 100% cases

Reply:
Tried, it breaks requests in different places